src/components/button/button.scss

```diff
@@ -18,6 +18,14 @@
     pointer-events: none;
 }
 
+button.mdc-button {
+    // As long as this component is depended on MDC,
+    // we need to force it to be font-agnostic.
+    // When MDC-dependency is removed, this block can also be removed.
+    font-family: inherit;
+    font-size: inherit;
+}
+
```

Comment:
I hope this pr: https://github.com/Lundalogik/lime-elements/pull/2668 gets merged before the current PR. Then we can revisit this block of code. Let's see which one goes in first.

Reply:
I'm guessing you've already revisited this since #2668 was merged?

Reply:
Ah, actually, maybe not?

<img width="834" alt="image" src="https://github.com/Lundalogik/lime-elements/assets/154725/f7c0e87e-e838-4bff-ab49-2f63cfc57453">

Reply:
Oh I forgot to update this comment. I've already done what was needed after merging https://github.com/Lundalogik/lime-elements/pull/2668

Not sure what the conflict is :/

Reply:
I fixed the conflict and pushed. Had nothing to to with that other PR. It was super weird why git couldn't already figure it out by itself. But fixed now.